src/test/ui/pattern/issue-71042-opaquely-typed-constant-used-in-pattern.rs

```diff
@@ -4,6 +4,7 @@
 fn main() {
     const C: impl Copy = 0;
     match C {
-        C | _ => {} //~ ERROR: opaque types cannot be used in patterns
+        C => {} //~ ERROR: `impl Copy` cannot be used in patterns
+        _ => {}
```

Comment:
What's with the change here?

Reply:
I wanted to make sure that we are reporting on the constant and not on the wild pattern

Reply:
You can do that with splitting `C | _` on two lines, no? IMO it's better not to change the test.